### PR TITLE
validator: add contact-info query to admin port

### PR DESCRIPTION
**Pull request by t-nelson**
#### Problem

No easy way to query all validator port assignments

#### Summary of Changes

Add `contactInfo` method to validator admin port
Add `contact-info` subcommand to `solana-validator`

### CODE REVIEW

File: validator/src/main.rs [hunk 1]
```diff
@@ -1809,6 +1821,41 @@ pub fn main() {
...
+                    exit(1);
+                });
+            if let Some(mode) = output_mode {
+                match mode {
+                    "json" => println!("{}", serde_json::to_string_pretty(&contact_info).unwrap()),
+                    "json-compact" => print!("{}", serde_json::to_string(&contact_info).unwrap()),
+                    _ => unreachable!(),
+                }
+            } else {
+                println!("Identity: {}", contact_info.id);
```
Comment (mvines) [inline validator/src/main.rs:1840]: Nit: impl `std::fmt::Display` for `AdminRpcContactInfo`?

**Review by mvines — APPROVED** [review]
nice idea!

File: validator/src/main.rs (hunk 1 above)
Comment (t-nelson) [reply validator/src/main.rs:1840]: Ah... you caught 1am Trent being lazy.  Will do!

Comment (t-nelson) [reply validator/src/main.rs]: done!

**Pull request merged**